src/Listeners/ClearNavTreeCache.php

```diff
@@ -0,0 +1,21 @@
+<?php
+
+namespace Rapidez\Statamic\Listeners;
+
+use Illuminate\Support\Facades\Cache;
+use Statamic\Events\NavTreeSaved;
+use Statamic\Eloquent\Structures\NavTree;
+
+class ClearNavTreeCache
+{
+    public function handle(NavTreeSaved $event): void
+    {
+        /** @var NavTree $tree */
+        $tree = $event->tree;
+        
+        $cacheKey = 'nav:' . $tree->handle() . '-' . config('rapidez.store');
+        if (Cache::has($cacheKey)) {
```

Comment:
I'm pretty sure this check isn't needed as forget already lets the respective driver check if it needs to remove it.
e.g. the file cache: https://github.com/laravel/framework/blob/5caf767f708aa6e8a3774104ecc7dd0471db805e/src/Illuminate/Cache/FileStore.php#L250